kratos/tests/cpp_tests/spaces/test_ublas_space.cpp

```diff
@@ -23,11 +23,11 @@
 namespace Kratos {
 namespace Testing {
 
-typedef UblasSpace<double, CompressedMatrix, boost::numeric::ublas::vector<double>> SparseSpaceType;
-typedef UblasSpace<double, Matrix, Vector> LocalSpaceType;
 
 KRATOS_TEST_CASE_IN_SUITE(UblasSpaceNormSparseMatrix, KratosCoreFastSuite)
 {
+    typedef UblasSpace<double, CompressedMatrix, boost::numeric::ublas::vector<double>> SparseSpaceType;
```

Comment:
Why not creating a specific `namespace UblasTesting` inside the `Testing` one to avoid declaring this in each test.

Reply:
i don't follow...how would u use it ?

Reply:
Maybe I did misunderstand the problem. As far as I understand you required to move the type declaration from the `Testing` namespace to each test because otherwise interferes with the new matrices test. If this is the case, we can create a namespace `UblasTests` and `KratosMatrixTests` within the `Testing` one to declare this once and encapsulate all the tests (I assumed this would work).